test/__snapshots__/test.js.snap

```diff
@@ -0,0 +1,98 @@
+exports["Serializer transform should generate module when the type is undefined"] = `/* This file was automatically generated. Do not edit by hand. */
```

Comment:
I cheated a little bit because the `chai-snapshot-matcher` plugin adds them all in one line. I manually changed that to be proper multiline strings